azure-pipelines.yml

```diff
@@ -23,13 +23,13 @@ jobs:
     vmImage: $(image)
   strategy:
     matrix:
-      'Ubuntu 20.04 LTS with GCC 10 (Debug, x86_64)':
-        image: ubuntu-20.04
-        analyzer: on
-        # not enabling "undefined" because of some false positives
-        sanitizer: address
-        cc: gcc-10
-        coverage: on
+      #'Ubuntu 22.04 LTS with GCC 12 (Debug, x86_64)':
```

Comment:
Maybe add a comment here that explains why this is disabled?

Reply:
Agree